discord/README.md

```diff
@@ -16,111 +16,3 @@ Populate `clientId` and `clientSecret` with the values from the Discord app's OA
 ```sh
 npm start
 ```
-
-## Cypress Troubleshooting
-
-> Reverse chronological order of appearance
-
-### Running against node 16.14.2
-
-```sh
-❯ npm start
-
-> @trshcmpctr/discord@1.0.0 start
-> node --experimental-specifier-resolution=node lib/index.js
-
-node:internal/errors:464
-    ErrorCaptureStackTrace(err);
-    ^
-
-Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'data-uri-to-buffer' imported from /Users/shanegarrity/dev/trshcmpctr/discord/node_modules/node-fetch/src/index.js
-    at new NodeError (node:internal/errors:371:5)
-    at packageResolve (node:internal/modules/esm/resolve:932:9)
-    at moduleResolve (node:internal/modules/esm/resolve:978:18)
-    at defaultResolve (node:internal/modules/esm/resolve:1080:11)
-    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
-    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
-    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
-    at link (node:internal/modules/esm/module_job:78:36) {
-  code: 'ERR_MODULE_NOT_FOUND'
-}
-```
-
-[nodejs/node#42116](https://github.com/nodejs/node/issues/42116) is similar.
-Points to a [symlinking issue](https://github.com/nodejs/node/issues/42195) introduced in 17.6.0 and backported in 16.14.x (?).
-Fix in [nodejs/node#xxx](https://github.com/nodejs/node/pull/42197) not backported to v16 until 16.15.0.
-
-Realizing that this is exactly what I was working around when I pinned the Drone images to 16.13.2.
-With `node:gallium`, everything broke on the next run of the pipeline after 16.14.0 was released as `latest`.
-
-The most current published Cypress image runs node 16.14.2.
-The next minor version available is only 16.5.0
-(see <https://github.com/cypress-io/cypress-docker-images/tree/master/included#cypressincluded>).
-
-There's probably no way to work around this unless I build my own image.
```

Comment:
just installing Cypress binary as a dependency now, not using any special Cypress image in github workflow

worth noting! node version in github workflow is controlled by `runs-on: ubuntu-22.04`